.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## PR Title :boom:
+
+Please title this PR with a summary of the change, preferably with the JIRA card number.
```

Comment:
I would like to give emphasis on format of `RHIROS-840`[CAPITAL-NO],

```suggestion
Please title this PR with a summary of the change, preferably with the JIRA card number[i.e RHIROS-840].
```

Reply:
Good stuff, changes updated, thanks!